app/views/account/users/_user.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td= link_to user.email, user.email
+  td= user.name
+  td
+    - if current_user.admin?
+      = simple_form_for(user, url: account_user_path(user)) do |f|
```

Comment:
ok, but next time if you decide to do something with user - implement on edit page.

PS: editing role - yes, easier, but how often do you think you will do it? and if you really want to do here, then do using ajax, and onchange event of select without page refresh (just an idea of what is easy to use, do not implement it)